### DISCUSSION TITLE
How to downgrade to 1.9.2 ?

### DISCUSSION THREAD

**Discussion by krauzac** [mrlt8/docker-wyze-bridge#659] [General]
What's the command to pull v1.9.2? 
 Something like (fill in the blank)
docker-compose pull image:wyze-bridge: ???

My Pi4 CPU is at 99% after pulling 1.10.1
If I stop it settles down to 30-40%
I have 4 cameras

**Comment by mrlt8** [comment]
hmm.. 99% from all four cores?

If you're using docker compose, then you should be able to edit your docker-compose.yml to something like this and restart the container:
```yaml
image: mrlt8/wyze-bridge:1.9.2
```

if you're using docker run, then just swap out the latest tag to 1.9.2 like:

```
docker run \
  -e WYZE_EMAIL=you@email.com \
  -e WYZE_PASSWORD=yourpassw0rd \
  -p 1935:1935 -p 8554:8554 -p 8888:8888 -p 5000:5000 \
  mrlt8/wyze-bridge:1.9.2
```

**Comment by krauzac** [comment]
Thanks!
